ui/src/util/usePagination.tsx

```diff
@@ -0,0 +1,11 @@
+import { useSearchParams } from "react-router-dom";
+
+export const usePagination = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const page = searchParams.get("page") ?? "";
```

Comment:
page_token

Reply:
This (and the others below are the parameter names used by the frontend, exposed to the user. But I guess we can keep them consistent with the backend.